### ISSUE TITLE
Windows Containers mode - networking issues

### ISSUE THREAD

**Issue by vaniovanio** [docker/for-win#1104]
### Expected behavior
Adding "fixed-cidr" : "192.168.137.0/24" should change the default "nat" IP address

### Actual behavior
Default "nat" IP address is still "Subnet": "172.27.96.0/20"
Because of this, containers don't get IP address.

### Information

  - Diagnostic ID from "Diagnose & Feedback" in the menu: 
A2225BC6-EC58-4F9C-95DD-23F28086A100/2017-09-19_16-35-54

Windows build is 1703, 15063.608 (Windows 10 Enterprise, 64-bit).

### Steps to reproduce the behavior
  1. Install Docker CE Stable
  2. Switch to Windows Containers
  3. Configure WinNAT using the following article: https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/user-guide/setup-nat-network#connect-a-virtual-machine
  4. Restart Docker and try to complete the following LAB: https://docs.microsoft.com/en-us/virtualization/windowscontainers/quick-start/building-sample-app
  5. On the step "Running the app", Docker fails on the "docker build -t myasp ." command, because nuget cannot connect to internet to restore packages...

**Comment by jasonbivins** [comment]
Hi @vaniovanio 
Have you tried changing the IP scheme through the GUI ?

![image](https://user-images.githubusercontent.com/28021664/30610329-d5583a2a-9d4b-11e7-97e8-873a4d630acc.png)

**Comment by JanneRantala** [comment]
@jasonbivins There's no network option when switched to Windows Containers? I've also opened an issue four months back about changing default network prefix [https://github.com/docker/for-win/issues/726](url), any ideas about that?

**Comment by vaniovanio** [comment]
Yes, as @JanneRantala correctly pointed out, there is no GUI for network configuration once the Windows Containers mode is enabled (something of a drawback). 

However, I was able to fix my networking issues and now the containers are connected to the host and the rest of the network. I fixed it by completely removing all traces of Docker installation and Hyper-V component in Windows 10. After I reinstalled everything from scratch the containers are now connected and working as expected.

The only issue that remains is the fact that the IP range, given to the containers is still not controlled by the "fixed-cidr" configuration... I want them to be with prefix 192.168.137.0/24, while they are with the default one 172.18.48.0/20...
